src/main/java/training/commands/Roller/stopRollerCommand.java

```diff
@@ -0,0 +1,18 @@
+package training.commands.Roller;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import training.subsystems.Roller.RollerSubsystem;
+
+public class stopRollerCommand extends Command {
+    private final RollerSubsystem roller = RollerSubsystem.getInstance();
+
+    public stopRollerCommand() {
+        addRequirements(roller);
+    }
+
+    @Override
+    public void execute() {
+        addRequirements(roller);
+        roller.stop();
+    }
+}
```

Comment:
this command will never end (not good)

Reply:
This command is meant to be used together with .withTimeout(something). Why is this not recommended?

Reply:
stop needs to stop and then end, at least this is what the name stop means